linode/manager
# M3-5661: Add Databases and Firewalls to Support Tickets

## Description

- Allows users to link **Databases** or **Firewalls** in a support ticket

## How to test

- Point Cloud to the alpha environment 
- Have a Database on your account
- Go to create a support ticket and link your database to the ticket
- Verify it was linked in alpha

Repeat the steps above for other entities such as Linodes and Firewalls to ensure I didn't break anything .
